python/educational-core/student/src/com/jetbrains/edu/learning/stepic/StepicUser.java

```diff
@@ -12,18 +12,26 @@
   private static final String STEPIC_SETTINGS_PASSWORD_KEY = "STEPIC_SETTINGS_PASSWORD_KEY";
   private static final Logger LOG = Logger.getInstance(StepicUser.class);
   private int id = -1;
-  private String myFirstName = "";
-  private String myLastName = "";
-  private String myEmail = "";
+  private String firstName = "";
```

Comment:
According to Idea code style field names start with "my" prefix. I see no reason in this rename